cloud/services/networkinterfaces/networkinterfaces_test.go

```diff
@@ -140,7 +141,7 @@ func TestReconcileNetworkInterface(t *testing.T) {
 				gomock.InOrder(
 					mSubnet.Get(context.TODO(), "my-rg", "my-vnet", "my-subnet").Return(network.Subnet{}, nil),
 					mLoadBalancer.Get(context.TODO(), "my-rg", "my-public-lb").Return(getFakeNodeOutboundLoadBalancer(), nil),
-					m.CreateOrUpdate(context.TODO(), "my-rg", "my-net-interface", matchers.DiffEq(network.Interface{
+					m.CreateOrUpdate(context.TODO(), "my-rg", "my-net-interface", gomockinternal.DiffEq(network.Interface{
```

Comment:
is it better to use gomockinternal? Asking because I've been using matchers.DiffEq everywhere so wondering if I should stop doing that

Reply:
The are the same thing. I moved an the `internal/matchers` package to `internal/matchers/gomock`, b/c I needed to make matchers for gomega too.